src/binaryutils.cpp

```diff
@@ -110,7 +110,7 @@ bool resizeNeeded(const cv::Mat& image, const std::shared_ptr<TensorInfo>& tenso
             rows = image.rows;
         }
     }
-    if (cols != Dimension(image.cols) || rows != Dimension(image.rows)) {
+    if ((!cols.match(Dimension(image.cols).getStaticValue())) || (!rows.match(Dimension(image.rows).getStaticValue()))) {
```

Comment:
Why do we need to convert static `image.cols` to Dimension object just to get static value back?
Wouldnt it be simpler to do:
```
cols.match(image.cols)
```